app/assets/javascripts/discourse/controllers/badges/show.js.es6

```diff
@@ -28,6 +28,11 @@ export default Ember.Controller.extend(BadgeSelectController, {
     return username ? userCount : modelCount;
   },
 
+  @computed("model.grant_count")
+  othersCount(grant_count) {
+    return grant_count - 1;
```

Comment:
Oh, this is not going to be a simple -1, you could have multiple badges of the type, so you need to subtract the number you have.

Reply:
Changed.
Should I consider that other people can also have multiple badges per person?